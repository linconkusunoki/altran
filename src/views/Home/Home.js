import React from 'react'
import { Link } from 'react-router-dom'

import Layout from '../../layout'

const Home = () => {
  return (
    <Layout.Wrapper>
      <Layout.Header />
      <Layout.Main>
        <h1>Home</h1>
        <ul>
          <li>
            <Link to="/gnome/1">gnome 1</Link>
          </li>
        </ul>
      </Layout.Main>
    </Layout.Wrapper>
  )
}

export default Home
