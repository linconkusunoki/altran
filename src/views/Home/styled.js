import styled from 'styled-components'

export const Title = styled.h1`
  font-size: 47px;
  color: #211338;
  font-family: 'Rancho';
  font-weight: 400;
  line-height: 1;
  margin: 0 0 10px;
  text-align: center;
`
