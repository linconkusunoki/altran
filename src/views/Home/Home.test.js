import React from 'react'
import { render } from '@testing-library/react'
import Home from './'

it('should render correctly', async () => {
  const { container } = render(<Home />)
  expect(container.firstChild).toMatchSnapshot()
})
