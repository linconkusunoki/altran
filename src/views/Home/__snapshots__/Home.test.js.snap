// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <main
    class="sc-bdVaJa dnnmyP"
  >
    <div
      class="sc-htpNat iiWFAZ"
    >
      <div
        class="sc-bwzfXH cnlxnu"
      >
        <h1
          class="sc-kkGfuU dYReHC"
        >
          Welcome to Brastlewark
        </h1>
        <p>
          Loading gnomes...
        </p>
      </div>
    </div>
  </main>
  <footer
    class="sc-bxivhb bkAgtj"
  >
    <div
      class="sc-htpNat iiWFAZ"
    >
      Crafted with some
       
      <span
        aria-label="made with love"
        role="img"
      >
        ðŸ§™
      </span>
       
      summoned by Lincon Kusunoki
    </div>
  </footer>
</div>
`;
